src/rust/inetstack/mod.rs

```diff
@@ -247,10 +247,13 @@ impl InetStack {
     pub fn listen(&mut self, qd: QDesc, backlog: usize) -> Result<(), Fail> {
         #[cfg(feature = "profiler")]
         timer!("inetstack::listen");
-        trace!("listen(): qd={:?} backlog={:?}", qd, backlog);
+        trace!("listen() qd={:?}, backlog={:?}", qd, backlog);
+
+        // FIXME: https://github.com/demikernel/demikernel/issues/584
         if backlog == 0 {
```

Comment:
I intentionally left code untouched -- added the reference to the concerning issue.

As I noted in the PR description, this is not possible because it breaks the legacy unit tests that bypass the PDPIX layer. Once we move Catnip and Catpowder as well we may introduce the logic there.

Note that this is the reason for which this PR does not close the original issue.

Reply:
Ah ok!